### DISCUSSION TITLE
Option to register daemon as a service

### DISCUSSION THREAD

**Discussion by andrey6498** [OpenTabletDriver/OpenTabletDriver#2235] [Ideas]
Seems that right now otd daemon is not supporting service mode(at least on windows).
Why running as service? Cuz this way you don't have to worry about running it manually while also minimizing delay after start(like if you have your tablet rotated 180° and use it as main input device that becomes somewhat annoying)

**Comment by jamesbt365** [comment]
You can already start OpenTabletDriver on startup, having it as a service is not exactly necessary, and the way that windows manages services isn't the greatest and therefore not always the best route. I don't see how this could be implemented without confusing regular users either, just having it creep in the background.

Handling services is also weird because its not something that can implemented the same way cross platform, it works on linux because the init system itself can just manage the process, with windows its not as straight forward.

**Comment by gonX** [comment]
Per-user services do exist on Windows, this might be interesting to have down the line to ensure availability in case the daemon crashes.

The upcoming rework (#2772) has enabling autostart built in and uses the startup folder for now.
